package/expo-package/src/optionalDependencies/deleteFile.ts

```diff
@@ -0,0 +1,25 @@
+let FileSystem;
+
+try {
+  FileSystem = require('expo-file-system');
+} catch (e) {
+  // do nothing
+}
+
+if (!FileSystem) {
+  console.log(
+    'expo-file-system is not installed. Installing this package will allow your users to save/delete file locally and access the cache dir for android/iOS.',
+  );
+}
+
+export const deleteFile = FileSystem
+  ? async ({ uri }: { uri: string }) => {
+      try {
+        await FileSystem.deleteAsync(uri, { idempotent: true });
+        return true;
+      } catch (error) {
+        console.log('File deletion failed...');
```

Comment:
Please include the `error` in this log. Otherwise, it gets quite challenging to figure out which file failed to be deleted.